src/app/pages/components/tree/input.component.html

```diff
@@ -0,0 +1,21 @@
+<form class="form" role="form" [formGroup]="form" (ngSubmit)="okClick()">
+    <fieldset>
+    <div class="modal-header">
+        <span>{{ modalHeader }}</span>
+        <button class="close" aria-label="Close" (click)="closeModal()">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+  
+      <div class="modal-body">
+       <div class=" form-group" >
+  <input type="text" placeholder="Name" class="form-control"  [formControl]="form.controls['node']" [ngClass] = "{'form-control-danger': form.controls['node'].hasError('required') && form.controls['node'].touched}"/>
```

Comment:
Also no need for the spaces in 
`[ngClass] = "{'form-control-danger': ...`
`[ngClass]="{'form-control-danger': ...`